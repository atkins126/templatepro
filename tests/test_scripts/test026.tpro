<input type="text" value="{{:phrasewithquotes}}">