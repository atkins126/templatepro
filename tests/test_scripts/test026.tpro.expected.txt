<input type="text" value="This &#34;and that&#34; with &#39;this and that&#39;">